README.md

```diff
@@ -47,7 +47,7 @@ When the SetLastArrayItem method is called, the following steps are taken:
 
     Let O be ? ToObject(this value).
     Let len be ? ToLength(? Get(O, "length")).
-    If len is zeo, then
+    If len is not zero, then
```

Comment:
maybe `> 0`?

Reply:
🤷‍♂️, how does the rest of the spec handle it?

Reply:
good point. I think it is, but I think that making implications explicit might still be helpful.

 - https://tc39.github.io/ecma262/#sec-string.raw uses `<= 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.includes uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.indexof uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.lastindexof uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.pop uses `is zero`
 - https://tc39.github.io/ecma262/#sec-array.prototype.reduce uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.reduceright uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.shift uses `is zero`

All the other use cases I found were part of a more generic algorithm, like a loop.

Sounds like `is 0` is the overarching convention. I'll make a PR to convert the remainder in the spec.

Reply:
https://github.com/tc39/ecma262/pull/1086

Reply:
Updated.